spec/builder_spec.rb

```diff
@@ -82,6 +82,15 @@ class Test < described_class
     end
   end
 
+  describe '.columns' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.